### DISCUSSION TITLE
Entry Server and Entry Client Missing

### DISCUSSION THREAD

**Discussion by emrekardaslar** [remix-run/remix#5952] [Q&A]
Hi everyone, entry server and entry client files are being hidden as I can see. How can I use those two files with remix verison: 1.14.3 and newer.

**Comment by emrekardaslar** [comment]
https://github.com/remix-run/remix/issues/4845 I saw that those files are optional for newer versions of remix but what should someone do if he/she wants to use those files, where can I learn that?

**Reply by kiliman** [reply]
run `npx remix reveal` and it will generate those files for you.

**Reply by emrekardaslar** [reply]
Thank you very much!
